README.md

```diff
@@ -41,17 +41,6 @@ In the File Sharing tab, click '+' icon and select `/files` (the directory you c
 
 ![screen shot 2017-02-09 at 12 49 54 am](https://cloud.githubusercontent.com/assets/720709/22776939/80c84282-ee66-11e6-9f6a-6d128af6ca06.png)
 
-Then, you need to comment out the first 4 lines of quickstart.sh or quickstartdev.sh
-
-These four lines needs to be commented out.
-
-```
- #docker-machine create --driver virtualbox default
```

Comment:
Well if people using old mac or old windows, which means if they use docker tool box, they need these lines... So I like to keep them.